examples/mixed_hmm/experiment.py

```diff
@@ -0,0 +1,196 @@
+import argparse
+import os
+import json
+import uuid
+import functools
+
+import torch
+
+import pyro
+import pyro.poutine as poutine
+
+import funsor.ops as ops
+from funsor.interpreter import interpretation
+from funsor.optimizer import apply_optimizer
+from funsor.sum_product import sequential_sum_product, sum_product
+from funsor.terms import lazy
+
+
+from model import model_sequential, model_parallel, guide_sequential
+from seal_data import prepare_seal
+
+
+def aic_num_parameters(model, guide=None):
+    """
+    hacky AIC param count that includes all parameters in the model and guide
+    """
+
+    def _size(tensor):
+        """product of shape"""
+        s = 1
+        for d in tensor.shape:
+            s = s * d
+        return s
+
+    with poutine.block(), poutine.trace(param_only=True) as param_capture:
+        model()
+        guide()
+
+    return sum(_size(node["value"]) for node in param_capture.trace.nodes.values())
```

Comment:
nit: use [numel()](https://pytorch.org/docs/stable/tensors.html#torch.Tensor.numel), as in `node["value"].numel()`